src/utilities/messages.ts

```diff
@@ -0,0 +1,8 @@
+import messages from '../config/constants/messages.json';
```

Comment:
This is very cool, but importing objects from `json` can be risky.

Instead define these constants in code:

```
type ErrorMessages = {
  'not-found-zdao': 'Not found zDAO';
  'empty-zdao-title': 'Empty zDAO title';
  'empty-gnosis-address': 'Empty Gnosis Safe address';
  'empty-gnosis-owners': 'Empty Gnosis Safe owners';
  'empty-voting-token': 'Empty Voting ERC20 Token address';
  'already-exist-zdao': 'zDAO already exists';
  'empty-metadata': 'Not found Token Transfer information';
  'not-gnosis-owner': 'Not a Gnosis Safe owner address';
  'not-implemented': 'Not implemented';
};

type ErrorTypes = keyof ErrorMessages;
```

and then change the function param to `(error: ErrorTypes )` and now intellisense will help you auto fill!